lib/wasmex.ex

```diff
@@ -188,8 +188,13 @@ defmodule Wasmex do
       rescue
         e in RuntimeError -> {false, e.message}
       end
+      
+    params = case return_value do
+      nil -> []
+      params -> [return_value]
```

Comment:
shouldn't this be either `nil` or "any other value"?

```suggestion
      _ -> [return_value]
```

☝️ I think this is why the elixir linter fails in CI